msgs/mavros_msgs/CommandTriggerInterval.srv

```diff
@@ -0,0 +1,7 @@
+# Type for controlling camera trigger interval and integration time
+
+float32   cycle_time			# Trigger cycle_time (interval between to triggers) - set to 0 to ignore command
+float32   integration_time	# Camera shutter integration_time - set to 0 to ignore command
+---
+bool success
+uint8 result
```

Comment:
No new line at end of file

Reply:
That srv file is a copy of original `/opt/ros/kinetic/share/mavros_msgs/srv/CommandTriggerInterval.srv`, so, will not make a change.